src/api.js

```diff
@@ -0,0 +1,15 @@
+import axios from 'axios';
+
+const baseURL = 'https://api.github.com/';
+
+export const getRepos = (query) => {
+  return axios.get(`${baseURL}search/repositories`, {
+    params: {
+      q: query
+    }
+  })
+  .then(res => res.data.items
+    .map(({ id, name, description, html_url }) => ({id, name,description, html_url })
```

Comment:
I'm not a fan of using lodash in general, but sometimes it could be better compared to big destructuring (shorter? more readable?)

```js
res => res.data.items.map(item => pick(item, [
  'id', 'name', 'description', 'html_url'
]))
```

in this specific case though, both solutions are fine